module_utils/kafka_manager.py

```diff
@@ -127,13 +128,30 @@ def create_topic(self, name, partitions, replica_factor,
         Creates a topic
         Usable for Kafka version >= 0.10.1
         """
+        self.create_topics([{
+            'name': name,
+            'partitions': partitions,
+            'replica_factor': replica_factor,
+            'options': config_entries,
+            'replica_assignment': replica_assignment
+        }])
+
+    def create_topics(self, topics, timeout=None):
+        """
+        Creates a topic
+        Usable for Kafka version >= 0.10.1
+        """
         if timeout is None:
             timeout = self.DEFAULT_TIMEOUT
         request = CreateTopicsRequest_v0(
             create_topic_requests=[(
-                name, partitions, replica_factor, replica_assignment,
-                config_entries
-            )],
+                topic['name'],
+                topic['partitions'],
+                topic['replica_factor'],
+                topic['replica_assignment']
+                if 'replica_assignment' in topic else [],
+                topic['options'].items() if 'options' in topic else []
```

Comment:
```suggestion
                topic.get('options', {}).items()
```

Reply:
Can't do that... Python does not allow that.